lib/views/widgets/template.dart

```diff
@@ -52,16 +52,16 @@ class Template extends StatelessWidget {
             ),
           ),
         ),
-          SafeArea(
-            child: Expanded(
-              flex: 1,
-              child: Container(
-                child: content,
-                alignment: Alignment.topLeft,
-                margin: const EdgeInsets.only(top: 60.0),
-              ),
+        SafeArea(
+          child: Expanded(
```

Comment:
remove expanded as a child of SafeArea since this returns an error ([reference](https://stackoverflow.com/questions/54905388/incorrect-use-of-parent-data-widget-expanded-widgets-must-be-placed-inside-flex))